src/components/templates/PosterPage/components/ShareModal/ShareModal.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback, useMemo } from "react";
+import { Modal } from "react-bootstrap";
+import { useCopyToClipboard } from "react-use";
+import classNames from "classnames";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFacebook, faTwitter } from "@fortawesome/free-brands-svg-icons";
+
+import { DEFAULT_MAP_BACKGROUND } from "settings";
+
+import { PosterPageVenue } from "types/venues";
```

Comment:
I expect we don't want to be importing `DEFAULT_MAP_BACKGROUND` here at all.

```suggestion
import { PosterPageVenue } from "types/venues";
```

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/08a1109cf3e296a0fc84656b5461df78f32fa60f